### PR TITLE
Aw gt star front

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Moves the stars to the image when there is one, and when there isn't an image style the stars correctly under the headline.

### CODE REVIEW

File: common/app/views/fragments/items/facia_cards/contentCard.scala.html [hunk 1]
```diff
@@ -192,7 +197,7 @@
             case _ => {}
         }
 
-        <div class="fc-item__content">
+        <div class="fc-item__content @if(item.starRating){fc-item__content--has-stars}">
```
Comment (regiskuckaertz) [inline common/app/views/fragments/items/facia_cards/contentCard.scala.html]: `if(item.starRating.isDefined)`?

File: static/src/stylesheets/module/facia-garnett/_item.scss [hunk 2]
```diff
@@ -492,24 +492,37 @@ $block-height: 58px;
 
 }
 
+//Base stars styling on facia cards *****
 .fc-item--type-review {
```
[the remark this answers is not in the document]
Comment (blongden73) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: Yes....just wanted to be sure.

File: static/src/stylesheets/module/facia-garnett/_item.scss [hunk 3]
```diff
@@ -492,31 +492,100 @@ $block-height: 58px;
...
+            width: .75em;
+
+            @include mq($from: desktop) {
+                height: 1em;
+                width: 1em;
+            }
+        }
+    }
+
+    .inline-star.star__item--golden {
```
Comment (sndrs) [inline static/src/stylesheets/module/facia-garnett/_item.scss]: just need `.star__item--golden`?

Comment (blongden73) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: done. :-)

File: static/src/stylesheets/module/facia-garnett/_item.scss [hunk 4]
```diff
@@ -492,31 +492,100 @@ $block-height: 58px;
...
+            height: .75em;
+            width: .75em;
         }
+    }
+}
 
-        .inline-star {
-            display: inline;
+.fc-item--has-image {
+    .fc-item__header {
```
Comment (sndrs) [inline static/src/stylesheets/module/facia-garnett/_item.scss]: it's it important that `fc-item__stars` sits below `fc-item__header` here? (np if so, if not, could we drop `fc-item__header`?)

File: static/src/stylesheets/module/facia-garnett/_item.scss [hunk 5]
```diff
@@ -492,31 +492,100 @@ $block-height: 58px;
...
+    }
+}
 
-            svg {
-                stroke: $garnett-neutral-1;
-                fill: transparent;
-                height: .8em;
-                width: .8em;
+.fc-item--list-mobile {
+    .fc-item__header {
```
Comment (sndrs) [inline static/src/stylesheets/module/facia-garnett/_item.scss]: as above

**Review by sndrs — APPROVED** [review]
couple of comments which would be good fix (if they are fixes) – but good to go if they're not relevent

File: static/src/stylesheets/module/facia-garnett/_item.scss (hunk 4 above)
Comment (blongden73) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: Yeah there are couple of scenarios where it's only the stars that sit within header that need to be hidden, and the stars that sit over the image will be visible. :-)

**Pull request merged**